schemas/v1/list-role-ids-response.yml

```diff
@@ -0,0 +1,21 @@
+$schema:  http://json-schema.org/draft-06/schema#
+title:                      "Get Role Ids Response"
+description: |
+  Get roleIds of roles based on continuationToken and limit
```

Comment:
"continuationToken and limit" may not make sense to people not familiar with the internals of Taskcluster. What do you think about "List of role IDs"?

Reply:
Right, that makes more sense!